tasks/test.js

```diff
@@ -75,45 +75,19 @@ module.exports = function (grunt) {
 
   grunt.registerTask('test:ui', [
     'checkPlugins',
-    'run:testEsServer',
```

Comment:
All references to the `run:testEsServer` tasks are gone, can we delete them also?

Reply:
hm. the one thing I'm concerned about is this option being passed:
```
        '-E', `discovery.zen.ping.unicast.hosts=localhost:${esTestConfig.getPort()}`,
```
I need to make sure that is still happening.

Reply:
Aha, I found it in something I wrote but forgot about: https://github.com/archanid/kibana/blob/archanid-18593/packages/kbn-test/src/es/es_test_cluster.js#L73

Reply:
I'm failing to see the connection between the grunt task of `run:testEsServer` and the `esTestConfig`